<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>488</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>DragonSettings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="1">
       <widget class="QSpinBox" name="ch1count">
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="ch2count">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>ch1 count</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>ch2 count</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="framelength">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1200</number>
        </property>
        <property name="maximum">
         <number>65520</number>
        </property>
        <property name="singleStep">
         <number>8</number>
        </property>
        <property name="value">
         <number>65520</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Frame length</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Frame count</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="framecount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>204800</number>
        </property>
        <property name="singleStep">
         <number>1024</number>
        </property>
        <property name="value">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ch1 shift</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="ch2amp">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ch2 amp</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ch2 shift</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="ch1shift">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>220</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="ch1amp">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="ch2shift">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ch1 amp</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Vertical axis scale</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QRadioButton" name="is8bits">
           <property name="text">
            <string>8 bit</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="is12bits">
           <property name="text">
            <string>12 bit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Channel</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QRadioButton" name="ch1en">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="ch2en">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
